blog/2024-04-23-dlc-channels-demystified.md

```diff
@@ -0,0 +1,317 @@
+This will become apparent in a latter section.
+
+The buffer transaction output can be spent in 3 different ways, but let's focus on two of them first.
+
+1.  CETs
+
+    The buffer transaction can be spent by one of several CETs.
+    For each CET, each party is able to provide their own signature for the buffer transaction output.
+    On the other hand, each party needs to be given the signature of their counterparty to be able to fulfill the 2-of-2 spend condition.
+
+    But we want the CETs to be spendable based on what the price of Bitcoin does, so signatures cannot be exchanged straight up.
+    Instead, we share "encrypted" signatures, more commonly known as *adaptor signatures*.
+    Adaptor signatures are incomplete signatures that can be completed by combining them with a certain secret.
+    And, importantly, they include a proof that they can be completed by that particular secret.
+
+    If we want a CET to only be spendable if the price of Bitcoin is `$X` at expiry time `T`, we construct its adaptor signature in such a way that it can only be unlocked if the oracle attests to price `$X` at time `T`.
+    This is possible because the oracle routinely announces the public random nonces[^5] it will use to attest to the price at specific times in the future.
+
+    After exchanging adaptor signatures for each CET, both you and the 10101 coordinator are ready to enforce the contract unilaterally based on the future attestation of the oracle.
+
+2.  Refund
+
+    By involving an oracle, we introduce an element of trust in a third party.
+    We can guarantee that the oracle knows how to unlock any of our CETs, but we cannot be sure that it will do so.
+    The oracle could misbehave, or it could simply be offline when we expect it to attest to a price.
+
+    If the oracle fails to meet the expectations, we fall back to an alternative *refund transaction*.
+    This refund transaction returns funds to their original owners, as if no financial contract was set up in the first place.
+
+    The refund transaction only exists to ensure that funds are not locked up forever.
+    It is only valid after a long timelock has expired.
+    In 10101, we use a 7-day long timelock from the time the buffer transaction is included in a block.
+
+    An unreliable oracle which causes DLCs to be refunded should not be trusted.
+    This is one of the reasons why it is preferable to use a quorum of oracles when setting up a DLC or a DLC channel[^6].
+
+
+Sealing the deal
+----------------
+
+With the setup out of the way, we just have to sign the fund transaction and put it on-chain.
+In 10101, you have to wait for one confirmation before you can do any further trading, but you can see how your position is performing right away.
+
+More trades
+-----------
+
+Imagine that you went long with your first position, but the price has tanked a bit.
+You might see this as an opportunity to double down.
```

Comment:
If you buy more, it doesn't sound like you are doubling down 🙃 or am i missing something 😅

Reply:
> strengthen one's commitment to a particular strategy or course of action, typically one that is potentially risky.
"he decided to double down and escalate the war"

Here the user would be doubling down on going long.